aws-sdk-core/lib/aws-sdk-core.rb

```diff
@@ -403,6 +403,21 @@ def use_bundled_cert!
       ))
     end
 
+    # Close any long-lived connections maintained by the SDK's internal
+    # connection pool.
+    #
+    # Applications that rely heavily on the `fork()` system call on POSIX systems
+    # should call this method in the child process directly after fork to ensure
+    # there are no race conditions between the parent process and its children
+    # for the pooled TCP connections.
```

Comment:
Should expand this to note that the method should return before making client calls in the child process. In theory, a threaded system might call this, start API calls before we clear, and have issues.